<?xml version="1.0" encoding="utf-8"?>
<rss version="2.0"><channel><title>PyVideo.org</title><link>https://pyvideo.org/</link><description></description><lastBuildDate>Sat, 18 Nov 2017 00:00:00 +0000</lastBuildDate><item><title>Unpacking an anomaly</title><link>https://pyvideo.org/pycon-ca-2017/unpacking-an-anomaly.html</link><description>&lt;h3&gt;Description&lt;/h3&gt;&lt;p&gt;This is a continuation of a series called Tales from the Ops Side. Back in November 2016, a seemingly innocuous piece of code that has been running for years starts throwing a very strange and nonsensical exception. This strange exception seemed to be thrown completely at random, could not be reproduced in any other environment, and would also self-heal! This talk tells the story of a baffling bug hunt spread over many weeks, involved all members of the company, including the CEO and CTO, and lead us deep into the guts of CPython and countless dead-ends.&lt;/p&gt;
</description><dc:creator xmlns:dc="http://purl.org/dc/elements/1.1/">Hany Fahim</dc:creator><pubDate>Sat, 18 Nov 2017 00:00:00 +0000</pubDate><guid isPermaLink="false">tag:pyvideo.org,2017-11-18:pycon-ca-2017/unpacking-an-anomaly.html</guid></item><item><title>Earthquakes and the Moon</title><link>https://pyvideo.org/pycon-ca-2015/earthquakes-and-the-moon-hany-fahim.html</link><description>&lt;h3&gt;Summary&lt;/h3&gt;&lt;p&gt;Timekeeping is no easy business. This talk tells the story of how earthquakes and even the Moon force the concept of the &amp;quot;leap second&amp;quot;, and the debacle in 2012 that took down most of the Internet. It also goes into some detail at how Python is handling this scenario.&lt;/p&gt;
</description><dc:creator xmlns:dc="http://purl.org/dc/elements/1.1/">Hany Fahim</dc:creator><pubDate>Sat, 07 Nov 2015 00:00:00 +0000</pubDate><guid isPermaLink="false">tag:pyvideo.org,2015-11-07:pycon-ca-2015/earthquakes-and-the-moon-hany-fahim.html</guid></item></channel></rss>